pkg/config/config.go

```diff
@@ -420,6 +420,11 @@ func InitConfig(config Config) {
 			config.SetDefault("procfs_path", "/proc")
 			config.SetDefault("container_proc_root", "/proc")
 		}
+		if pathExists("/host/sys") {
```

Comment:
This will set `HOST_SYS` for all agent process run in a container and not just the process-agent. Are we OK with that ? Have we tested the possible impact ?

Reply:
That's a good point - I didn't realize that the core agent uses gopsutil until you had mentioned it. I will move this back to the process agent.